src/AggregateSpecsPlugin.ts

```diff
@@ -11,6 +11,7 @@ import {
   MONEY_OID,
   NUMERIC_OID,
 } from "./interfaces.js";
+import { EXPORTABLE } from "./EXPORTABLE.js";
```

Comment:
Should we be including this or should we be grabbing EXPORTABLE from build, like below? I've seen both.

Reply:
If you can get it from `build` you should, the `EXPORTABLE` import is only to support phases where `build` does not exist (i.e. the `gather` phase). However, do NOT call it as `build.EXPORTABLE(...)`; you must ensure it's a standalone value (`const { EXPORTABLE } = build;` then `EXPORTABLE(...)`) for the lint rules to work correctly.